### DISCUSSION TITLE
Everything gone

### DISCUSSION THREAD

**Discussion by matrixsreal1** [iotaledger/firefly#2916] [Help & Support]
Hello,
my PC crashed so that i had no longer access to my Firefly.
So i got a new PC reinstalled firefly - restored my account, then created the new profile but now everything is gone - just look like a new account without anything on it?

What did i wrong or missed i something?

Thanks for help!

**Comment by MiKeF05** [comment]
How did you restore your account? Do you use a ledger profile or software profile?

**Comment by matrixsreal1** [comment]
Hey, 

restored with the 24 words. i don't use a ledger profile, what exactly is a software profil?

**Reply by MiKeF05** [reply]
Software profile is when you have 24 words generated by Firefly. Can you go to settings and try the balance finder? If you have many addresses you might need to search a few times in a row without closing the balance finder popup.

**Comment by matrixsreal1** [comment]
Well i tried out that search option, buut it nearly takes a second and then its done without finding anything...

I dont really understand how it works that "profile" seems to be a local thing, and i dont login anywhere - the only thing that shows me we are not talking about an only local thing it that generated 24 words thing

But i dont need to login, i had a profile with currency on it and now when i reinstalled it i just need to create a "new" profile? From where should firefly know that i have currency, that i had an account?

I dont understand and i dont have any access to my wallet any more...

Some further suggestions?

**Reply by MiKeF05** [reply]
First of all the information about your balance is stored on nodes in the network. Firefly connects to a node and fetches these information. The balance information is not stored locally. So you need a internet connection that Firefly is able to get that info.

So you created a complete new profile with new 24 words? Or did you restore your profile with your previous 24 words? Do not share them to anyone as the owner of the 24 words is able to transfer the funds.
